PullRequestR/android/app/src/test/java/com/example/bradygaster/pullrequestr/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package com.example.pullrequestr;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
```

Comment:
Remove this test, since I don't think we want to go and write a bunch of real unit tests :)